CONTRIBUTING.md

```diff
@@ -42,6 +42,10 @@ Our issue tracker utilizes several labels to help organize and identify issues.
 - `bug` - A bug is a _demonstrable problem_ that is caused by the code in the repository. Please check if the issue has already been reported before reporting a new bug.
 - `duplicate` - A duplicate signifies that another issue encapsulates or restates this issue. Duplicates will be closed and referenced as such.
 - `enhancement` - An enhancement is a feature request. Please provide as much detail as possible and consider whether your idea fits the scope and aims of this project.
+- `breaking change` - this issue warrants a major release and potentially changes APIs for downstream consumers.
+- `chore` - this issue improves internal tooling or build processes only.
+- `documentation` - this issue affects documentation only.
+- `polish` - this issue affects CSS or has stylistic changes.
```

Comment:
Hmm... I think CSS and stylistic changes are more than polish in this repo.

Reply:
substituted `css` for `polish`... i agree that it would be confusing. `css` label is used in Core.